docs/resources/object_bucket.md

```diff
@@ -118,6 +118,10 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the bucket.
 * `tags` - (Optional) A list of tags (key / value) for the bucket.
+
+* ~> **Important:** The Scaleway console does not support `key/value` tags yet, so only the tags' values will be displayed.
+But keep in mind that if you make any change to your bucket's tags using the console, it will overwrite them with the format `value/value`.
```

Comment:
```suggestion
Keep in mind that if you make any change to your bucket's tags using the console, it will overwrite them with the format `value/value`.
```